### DISCUSSION TITLE
Penalization for falsely flagged Pokemon

### DISCUSSION THREAD

**Discussion by imeangay** [kwsch/PKHeX#3670] [Q&A] [answered]
I've used PKHeX before for EV and IV stuff, never had issues, I am skeptical with the invalid flagging it has done for some of my Pokémon in Scarlet and Violet, but I want to edit the EV's for some of my Pokémon.  
Have any of you editing the EV's of your Pokémon and been flagged for the invalid checks the tool has given you? I got about 6 Pokémon that are flagged as invalid encounters, I'm paranoid that if I inject it into my game even though I did not do anything to flag illegal, that the illegal tagged Pokémon the tool says are invalid will somehow flag my game.

**Comment by sora10pls** [comment]
You are likely editing encounters that originated from Tera Raid Battles. Editing any of these encounters’ immutable stats (PID, Height/Weight/Scale, IVs, Encryption Constant, etc) will accurately flag them as illegal. All of these stats are derived from a seed when generating the Pokémon, so those stats can not be changed without making them illegal (no correlation after your edits).


You can edit more lenient encounters to have custom IV spreads, such as Eggs, which have less restrictions.

**Reply by imeangay** [reply]
No no what I mean is currently in my boxes I have pokemon both caught in raids, to pokemon that were just caught randomly such as galado mountains for my Galalie are marked as invalid "can't match valid encounter" I'm worried that because the tool flags them that they will somehow get marked as illegal in the game, thus im worried about injecting a save period based on the invalid encounters the tool is telling me I have.

Im worried that the pokemon the tool says are illegal, will someone actually become illegal and flag my game if I inject my save back into my game, regardless if I do or do not edit anything, am I just being paranoid?

**Comment by Lusamine** [comment, marked as answer]
If you caught it legitimately, it is legitimate, regardless of what any tool or person says.
